src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicClientTransformer.java

```diff
@@ -92,12 +83,10 @@
 
   private static final String XAPI_TEMPLATE_FILENAME = "csharp/gapic_client.snip";
```

Comment:
Nit: Rename `XAPI`. (You know you wanna 😛 )

Reply:
yes I do indeed!